runtime/src/lib.rs

```diff
@@ -523,32 +578,50 @@ construct_runtime!(
         Aura: pallet_aura = 23,
         AuraExt: cumulus_pallet_aura_ext = 24,
 
+        // Scheduler
+        Scheduler: pallet_scheduler = 28,
+
+        // Preimage
+        Preimage: pallet_preimage = 29,
+
         // XCM helpers.
         XcmpQueue: cumulus_pallet_xcmp_queue = 30,
         PolkadotXcm: pallet_xcm = 31,
         CumulusXcm: cumulus_pallet_xcm = 32,
         MessageQueue: pallet_message_queue = 33,
 
+        // Contracts
         Contracts: pallet_contracts = 40,
+
+        // Nfts
+        Nfts: pallet_nfts = 50,
+        NftFractionalization: pallet_nft_fractionalization = 51,
+        Assets: pallet_assets::<Instance1> = 52,
     }
 );
 
 #[cfg(feature = "runtime-benchmarks")]
 mod benches {
     frame_benchmarking::define_benchmarks!(
         [frame_system, SystemBench::<Runtime>]
+        [pallet_assets, Local]
```

Comment:
Curious why this is then also not `TrustBacked`. I see this is also how its done for asset hub so no need to change. 

I'm just always curious why they called local assets `TrustBackedAssets` in stead of `LocalAssets`. Foreign assets on the local chain are in practise also local assets so perhaps that is why they overcomplicated the name. Just had to share this :)